README.md

```diff
@@ -0,0 +1,4 @@
+# cluster-api-provider-openshift
+
+This repository contains Cluster API provider implementations for the OpenShift container platform.
```

Comment:
Is it worth mentioning we are implementing bootstrap and controlplane type of CAPI providers here?

Reply:
I think we can update this later, until we have something tangible I had planned to keep this relatively light